Model/User.php

```diff
@@ -132,10 +132,12 @@
      * Sets the creation date.
      *
      * @param \DateTime|null $createdAt
+     * @return $this
      */
     public function setCreatedAt(\DateTime $createdAt = null)
     {
         $this->createdAt = $createdAt;
+        return $this;
```

Comment:
[Please, add a blank line before ```return``` statements, unless the return is alone inside a statement-group](
https://symfony.com/doc/current/contributing/code/standards.html#structure).